train/train.py

```diff
@@ -138,6 +138,11 @@ def train():
 
     ## main loop
     coord = tf.train.Coordinator()
+    threads = []
+    for qr in tf.get_collection(tf.GraphKeys.QUEUE_RUNNERS):
+        threads.extend(qr.create_threads(sess, coord=coord, daemon=True,
+                                         start=True))
+
```

Comment:
Thank your!!! 

I just intended to add a data prefecth queue manually, but didn't know how yet.